core/src/main/java/com/yahoo/oak/ValueUtilsImpl.java

```diff
@@ -60,47 +22,49 @@ public static void setLengthFromOffHeap(Slice s){
             T transformation = transformer.apply(value);
             return result.withValue(transformation);
         } finally {
-            unlockRead(value.s);
+            value.s.unlockRead();
         }
     }
 
     @Override
     public <V> ValueResult put(Chunk<?, V> chunk, ThreadContext ctx, V newVal, OakSerializer<V> serializer,
                                MemoryManager memoryManager, InternalOakMap internalOakMap) {
```

Comment:
`MemoryManager memoryManager,` can be removed here as well.

Reply:
removed